liip/imagine-bundle/1.7/etc/routing/imagine.yaml

```diff
@@ -0,0 +1,19 @@
+liip_imagine_filter: 
```

Comment:
is it possible inclide `@LiipImagineBundle/Resources/config/routing.xml` instead of copy-pasting it content here?